source/manual/howto-backup-and-restore-in-aws-rds.html.md

```diff
@@ -230,13 +230,26 @@ This requires updating the CNAME for `local-links-manager-postgres`.
 4. Replace the value with the new RDS backup and save your changes. This takes about 60 seconds, you can click "view status" for updates. Once updated it will say `INSYNC`.
 5. SSH back into the machine and query for the record you deleted. If the record is back this should verify the app is now using the backup database.
 
-### 3. Connect back to the original database
+> This is only a temporary solution, to be used in an incident. You should continue onto the next section for a permanent solution.
+
+## Ensure your setup will continue to work if infrastructure is reprovisioned
+
+If new infrastructure is provisioned, then the "Point the application to the new RDS instance" solution above will break, as Terraform would fall out of sync with the manual changes. We either need to update Terraform with our changes, or manually get our infrastructure back to how it used to be.
+
+*For the purposes of drilling*, it's quicker and easier to do the latter. Simply repeat the steps to [point the application to the new RDS instance](#point-the-application-to-the-new-rds-instance), but this time connect to the original database. If you need to find the endpoint again for the original database, navigate to Amazon RDS, find the database in the list and look for `Endpoint & Port` under the `Connectivity & Security` tab.
+
+*If this is not a drill*, then you would not want to connect to the original database again - you've created a database from a backup for a reason!
+The safest approach would be to update Terraform to refer to the new database.
 
-Repeat the steps covered in [Step 5](#connect-to-the-restored-backup-database).
+Alternatively, you could:
 
-> If you need to find the endpoint again for the original database, navigate to Amazon RDS, find the database in the list and look for `Endpoint & Port` under the `Connectivity & Security` tab.
+1. [Delete the original database](#delete-an-obsolete-database)
```

Comment:
Would we want to carry out this step last, after the Terraform is updated?

Reply:
In this numbered list of instructions, we're not touching any Terraform.
We'd need to delete the original database first because we'll then be creating a new database under the same hostname (can't have two databases with the same name). Unless I've misunderstood how it works? 🤔

Reply:
Ah yes that makes sense!